Tests/MapboxNavigationTests/FeedbackViewControllerSnapshotTests.swift

```diff
@@ -0,0 +1,79 @@
+import XCTest
+import Foundation
+import MapboxDirections
+import SnapshotTesting
+@testable import TestHelper
+@testable import MapboxNavigation
+@testable import MapboxCoreNavigation
+
+class FeedbackViewControllerSnapshotTests: TestCase {
+    
+    var feedbackViewController: FeedbackViewController!
+    
+    override func setUp() {
+        super.setUp()
+        isRecording = false
+        
+        feedbackViewController = FeedbackViewController(eventsManager: NavigationEventsManagerSpy(), type: .passiveNavigation)
+        DayStyle().apply()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        feedbackViewController = nil
+    }
+    
+    func createDetailedFeedbackViewController() -> FeedbackSubtypeViewController? {
+        guard feedbackViewController.sections.count > 0, let feedback = feedbackViewController.currentFeedback else {
+              XCTFail("Failed to create detailed FeedbackViewController.")
+            return nil
+          }
+        let item = feedbackViewController.sections[0]
+        let detailedFeedbackViewController = FeedbackSubtypeViewController(eventsManager: NavigationEventsManagerSpy(),
+                                                                           feedbackType: item.type,
+                                                                           feedback: feedback)
+        return detailedFeedbackViewController
+    }
+    
+    func testDayFeedbackViewController() {
+        assertImageSnapshot(matching: feedbackViewController, as: .image(precision: 0.95))
+    }
+    
+    func testNightFeedbackViewController() {
+        UILabel.appearance(whenContainedInInstancesOf: [FeedbackViewController.self]).backgroundColor = .black
+        UILabel.appearance(whenContainedInInstancesOf: [FeedbackViewController.self]).textColor = .white
+        FeedbackStyleView.appearance(whenContainedInInstancesOf: [FeedbackViewController.self]).backgroundColor = .black
+        FeedbackCollectionView.appearance().backgroundColor = .black
+        FeedbackCollectionView.appearance().cellColor = .white
+        assertImageSnapshot(matching: feedbackViewController, as: .image(precision: 0.95))
+    }
```

Comment:
This doesn't necessarily verify that default Navigation SDK styling works as expected. I think it'd make more sense to either create own style that inherits from `NightStyle` and then apply it, or use it directly.

Reply:
Thanks for the suggestion, the class that inherits from `NightStyle` would make more sense for testing purpose. And for the case to test the default appearance,  currently we have no method to clear out the style. Should we add one for the `DayStyle`?

Reply:
https://github.com/mapbox/mapbox-navigation-ios/issues/3979 will be used to track the style restore and default UI appearance testing. This Pr will be responsible for the testing when `MaoboxNavigation.Style` applied.